docs/design/coreclr/botr/vectors-and-intrinsics.md

```diff
@@ -156,6 +156,14 @@ private void SomeVectorizationHelper()
 }
 ```
 
+#### Non-Deterministic Intrinsics in System.Private.Corelib
+
+Some APIs exposed in System.Private.Corelib are intentionally non-deterministic across hardware and instead only ensure determinism within the scope of a single process. To facilitate the support of such APIs, the JIT defines `Compiler::BlockNonDeterministicIntrinsics(bool mustExpand)` which should be used to help block such APIs from expanding in scenarios such as ReadyToRun. Additionally, such APIs should recursively call themslves so that indirect invocation (such as via a delegate, function pointer, reflection, etc) will compute the same result.
```

Comment:
```suggestion
Some APIs exposed in System.Private.Corelib are intentionally non-deterministic across hardware and instead only ensure determinism within the scope of a single process. To facilitate the support of such APIs, the JIT defines `Compiler::BlockNonDeterministicIntrinsics(bool mustExpand)` which should be used to help block such APIs from expanding in scenarios such as ReadyToRun. Additionally, such APIs should recursively call themselves so that indirect invocation (such as via a delegate, function pointer, reflection, etc) will compute the same result.
```

Reply:
Fixed.